mkosi.sanitizers/mkosi.postinst

```diff
@@ -56,6 +56,7 @@ wrap=(
     id
     integritysetup
     iscsid
+    knotc
```

Comment:
Based on my experiments `lsof`, `keymgr` and `knot` has to be wrapped as well. I don't think this sort of tweaking is maintainable though.

Reply:
With those things wrapped it fails to start knot though
https://github.com/evverx/systemd/actions/runs/13277825345/job/37070536901
```
TEST-75-RESOLVED.sh[163]: + mkdir -p /run/knot
TEST-75-RESOLVED.sh[163]: + chown -R knot:knot /run/knot
TEST-75-RESOLVED.sh[163]: + systemctl start knot
[FAILED] Failed to start knot.service - Knot DNS server.
[FAILED] Failed to start TEST-75-RESOLVED.service - TEST-75-RESOLVED.
```
I didn't dig deeper.